phpBB/phpbb/install/module/install_data/task/add_languages.php

```diff
@@ -103,6 +109,16 @@ public function run()
 		$insert_buffer->flush();
 	}
 
+	private function get_terms_of_use_from_lang($lang_iso)
+	{
+		global $phpEx, $phpbb_root_path;
+		$lang = array();
+
+		include($phpbb_root_path . 'language/' . $lang_iso . "/ucp." . $phpEx);
+		return $lang['TERMS_OF_USE_CONTENT'];
+
```

Comment:
Same as above.

Reply:
Also, couldn't you use the language class here? And please do not use the `global`s, as I'm fairly certain that these could be injected into the class.